lib/server.js

```diff
@@ -452,6 +452,80 @@ function createServer(options, clients, name) {
         log: log
     });
 
+    //
+    // This block exists to run HEAD errors through translateError like we do
+    // for other errors (those via formatters).
+    //
+    // When we're about to return an error due to a failure of a handler, we'll
+    // get the 'restifyError' event from:
+    //
+    // https://github.com/restify/node-restify/blob/6.x/lib/server.js#L1283
```

Comment:
Generally I'm not a fan of line numbers in code comments... but these references seem unlikely to change for the time horizon we're looking at. The 6.x branch hasn't changed in almost three years.

Reply:
Yeah, that makes sense. The reason I left it in is that as you pointed out it hasn't changed for 3 years and this branch is no longer maintained. Even the 7.x version is listed to only get backports for critical issues: https://github.com/restify/node-restify#supported-node-versions so I think I'll leave it for now and take my chances that some future programmer–archaeologist will find this message and be shaking their head at me. Hello from 2020, and sorry!